app/views/liking_relations/destroy.turbo_stream.erb

```diff
@@ -0,0 +1,8 @@
+<%= turbo_stream.update "post-#{@post.id}-points" do %>
+    <%= pluralize(@post.points, "point") %>
+<% end %>
+
+<% data = { method: :post, class: 'text-secondary' } %>
+<%= turbo_stream.update "post-#{@post.id}-upvote" do %>
+    <%= render 'posts/upvote_form', post: @post, data: data %>
+<% end %>
```

Comment:
To wygląda prawie tak samo jak plik create.turbo_stream.erb. Chyba przydałoby się wyrzucić to do partial'a.